ddtrace/internal/context_manager.py

```diff
@@ -2,18 +2,20 @@
 import threading
 from ddtrace.vendor import six
 
+from ddtrace.compat import contextvars
 from .logger import get_logger
 from ..context import Context
 
 log = get_logger(__name__)
 
-try:
-    from contextvars import ContextVar
+_DD_CONTEXTVAR = contextvars.ContextVar("datadog_contextvar", default=None)
 
-    _DD_CONTEXTVAR = ContextVar("datadog_contextvar", default=None)
-    CONTEXTVARS_IS_AVAILABLE = True
+try:
+    import contextvars
 except ImportError:
     CONTEXTVARS_IS_AVAILABLE = False
+else:
+    CONTEXTVARS_IS_AVAILABLE = True
```

Comment:
do we need this for anything anymore? it should always be available now, no?

Reply:
Yes, but we still need to know whether it "really" is available because that let's us know whether support exists in other libraries (like asyncio) in order then to patch additional context propagation code: https://github.com/DataDog/dd-trace-py/blob/e9b5798da0bba2aa45459e74a2ac205d7fa68461/ddtrace/contrib/asyncio/patch.py#L19-L22

Simply backporting contextvariables doesn't make it work with other libraries, we still need to instrument them to support contextvariables